src/lavinmq/launcher.cr

```diff
@@ -89,6 +91,26 @@ module LavinMQ
       Log.info &.emit("logger settings", level: @config.log_level.to_s, target: target)
     end
 
+    private def setup_log_exchange
+      queue_name = "logstream"
+      exchange_name = "amq.lavinmq.log"
+      vhost = @amqp_server.vhosts["/"]
+      vhost.declare_exchange(exchange_name, "topic", true, true, true)
+      vhost.declare_queue(queue_name, true, false)
+      vhost.bind_queue(queue_name, exchange_name, "#")
+      log_channel = ::Log::InMemoryBackend.instance.add_channel
+      spawn do
+        while entry = log_channel.receive
+          vhost.publish(msg: Message.new(
+            exchange_name,
+            entry.severity.to_s,
+            "#{entry.timestamp} [#{entry.severity.to_s.upcase}] #{entry.source} - #{entry.message}",
```

Comment:
The user might want to format the message differently, with the timestamp and the severity, no need to include it in the message body as it's available by other means.
```suggestion
            "#{entry.source} - #{entry.message}",
```